package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack",
-    "start:dev": "webpack-dev-server",
+    "start:": "webpack-dev-server",
```

Comment:
![image](https://github.com/ANTOINE1128/To-Do-List/assets/93039359/b633cbdc-2d9a-4546-9b09-26f786a88462)

- [ ] Great work on implementing the requested changes so far but the issue is still there 😨 as you can see in the image above 🤔 Kindly remove the colon `:` which is highlighted in the above to make your app work as expected 👍